src/ServicePulse.Host/app/lib/bootstrap-3.3.5/css/bootstrap.css

```diff
@@ -1583,8 +1583,8 @@ pre code {
   overflow-y: scroll;
 }
 .container {
-  padding-right: 15px;
-  padding-left: 15px;
+  padding-right: 0;
+  padding-left: 0;
```

Comment:
We can override those settings i particular.css (set paddings to 0). I verified it and it works. We could update bootstrap when we would want to benefit of any of the bug fixes or new features that they release. For 3.3.6 its: 
https://github.com/twbs/bootstrap/releases/tag/v3.3.6

What is more, we can decide to start using minified css which we would have to update with those values. I find it risky to modify 3rd party libraries. I am happy to override the styles if you don't mind